kamon-akka-2.4.x/src/main/scala/kamon/akka/instrumentation/ActorInstrumentation.scala

```diff
@@ -74,7 +75,13 @@ class ActorCellInstrumentation {
   def aroundReplaceWithInRepointableActorRef(pjp: ProceedingJoinPoint, unStartedCell: UnstartedCell, cell: Cell): Unit = {
     // TODO: Find a way to do this without resorting to reflection and, even better, without copy/pasting the Akka Code!
     val unstartedCellClass = classOf[UnstartedCell]
-    val queueField = unstartedCellClass.getDeclaredField("akka$actor$UnstartedCell$$queue")
+
+    val queueFieldName = Properties.versionNumberString.split("\\.").take(2).mkString(".") match {
```

Comment:
this could be a val on a companion object of `ActorCellInstrumentation` given that whatever we resolve, will never change. In fact, now that I think about it, the whole reflection thing could be resolved once in a companion object and then have a single helper function there to get the queue member from the unStartedCell. The member selection is still ugly, but will run only once :)